config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  resources :users, only: [:index, :show] do
-    resources :posts, only:[:index, :show]
-  end
+  root to: 'users#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
-  root "users#index"
-end
+  resources :users do
+    resources :posts do
+      resources :comments, :likes
```

Comment:
- [ ] [OPTIONAL] Please always ensure to only expose the necessary routes. In this case, only the `new` and `create` routes should have been exposed for the `posts` and `comments`, while only the `create` route should have been exposed for the `likes`. The others are not needed at this point 👍 
**HINT:** This is a great way to do the above
```ruby
resources :posts, only: %i[new create]
  resources :comments, only: %i[new create]
```